src/coloring/high_level.jl

```diff
@@ -1,9 +1,9 @@
-abstract type ColoringAlgorithm end
-struct GreedyD1Color <: ColoringAlgorithm end
-struct BacktrackingColor <: ColoringAlgorithm end
-struct ContractionColor <: ColoringAlgorithm end
-struct GreedyStar1Color <: ColoringAlgorithm end
-struct GreedyStar2Color <: ColoringAlgorithm end
+abstract type SparseDiffToolsColoringAlgorithm <: ArrayInterface.ColoringAlgorithm end

```

Comment:
`SparseDiffToolsColoringAlgorithm` not a fan of the package name plugged into types, C-like. Couldn't we not-export `ColoringAlgorithm`, thus calling it with `SparseDiffToolsColoringAlgorithm`. Other solution would be to remove the abstract type all together and use `ArrayInterface.ColoringAlgorithm` directly